### PR TITLE
Adding onnx2torch

### PR DESCRIPTION
**Pull request by senysenyseny16**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by senysenyseny16** [comment]
@conda-forge-admin, please ping team

File: recipes/onnx2torch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Nice Onnx to Pytorch converter"
+  doc_url: https://github.com/ENOT-AutoDL/onnx2torch
+  dev_url: https://github.com/ENOT-AutoDL/onnx2torch
+
+extra:
+  recipe-maintainers:
+    - ENOT-AutoDL
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:47]: The submitter of the recipe should be the maintainer.

File: recipes/onnx2torch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:5]: No need to quote all the jinja variables.

File: recipes/onnx2torch/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f668f6ecb2f34029d4c85a66be2f4a329a3d5d2ed8513395e508322d20dcb859
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:15]: This can probably be noach if the failure on Windows is just a missing dependency.

**Review by ocefpaf — CHANGES_REQUESTED** [review]

File: recipes/onnx2torch/meta.yaml (hunk 1 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: :heavy_check_mark:

File: recipes/onnx2torch/meta.yaml (hunk 2 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: :heavy_check_mark:

File: recipes/onnx2torch/meta.yaml (hunk 3 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: :heavy_check_mark:

Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: @ocefpaf I added `noarch: python` but build is failed on win. What have I done wrong?

Comment (ocefpaf) [reply recipes/onnx2torch/meta.yaml]: Like I mentioned above, if the failure is a missing dependency, they it is fine. What failed?

Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: There is no `torchvision`:
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("torchvision[version='>=0.9.0']")}
```

Comment (ocefpaf) [reply recipes/onnx2torch/meta.yaml]: Yep. So it is fine to leave it as noarch.

Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: Okay, thanks for your help.

**Comment by senysenyseny16** [comment]
@conda-forge-admin, please ping team

File: recipes/onnx2torch/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - pip
+    - python >=3.6
+    - torchvision >=0.9.0
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:23]: Unless the package imports itself at build time you don't need the run time dependencies on host.

```suggestion
    - pip
    - python >=3.6
```

File: recipes/onnx2torch/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - python >=3.6
+    - torchvision >=0.9.0
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:28]: There is a direct dependency on `pytorch >=1.8.0`.

```suggestion
    - pytorch >=1.8.0
    - torchvision >=0.9.0
```

PS: next time try to create the recipe with grayskull. It makes a recipe that is almost "merge-ready."

File: recipes/onnx2torch/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - onnx2torch
+    - onnx2torch.node_converters
+    - onnx2torch.utils
```
Comment (ocefpaf) [inline recipes/onnx2torch/meta.yaml:34]:
```suggestion
    - onnx2torch.utils
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/onnx2torch/meta.yaml (hunk 4 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml]: :heavy_check_mark:

File: recipes/onnx2torch/meta.yaml (hunk 5 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml:25]: Okay. It was created by `conda skeleton pypi onnx2torch`.

```python
   - pytorch >=1.8.0
   - torchvision >=0.9.0
```
It didn't work on osx build test due to inconsistent versions of pytorch and torchvision, but ok, i revert it.

File: recipes/onnx2torch/meta.yaml (hunk 6 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml:31]: :heavy_check_mark:

File: recipes/onnx2torch/meta.yaml (hunk 5 above)
Comment (senysenyseny16) [reply recipes/onnx2torch/meta.yaml:26]: Works now, thanks!

**Pull request merged**